test/ApplicationTest.php

```diff
@@ -142,12 +143,14 @@ public function testPipeCanAcceptAPathArgument($middleware): void
             ->method('pipe')
             ->with(new PathMiddlewareDecorator('/foo', $preparedMiddleware));
 
-        $this->assertNull($this->app->pipe('/foo', $middleware));
+        $this->app->pipe('/foo', $middleware);
     }
 
     public function testPipeNonSlashPathOnNonStringPipeProduceTypeError(): void
     {
-        $middleware1 = static fn(RequestInterface $req, ResponseInterface $res): ResponseInterface => $res;
+        /** @psalm-suppress UnusedClosureParam */
```

Comment:
We can just drop the parameters, no?

Reply:
I prefer the suppression here over the loud complaints from psalm - `MiddlewareParam` types the possible callable with the usual middleware signature